### DISCUSSION TITLE
Create virtual devices

### DISCUSSION THREAD

**Discussion by shanedevane** [hbldh/bleak#596] [Q&A] [answered]
hi, newbie question, can Bleak create virtual devices?
I want to simulate many virtual devices on my windows 10 machine - so that my iPhone/Android app sees multiple devices.
Can Bleak do that?
Sorry for the newbie question; I read the docs but couldn't find anything suitable (but i might be using the wrong terms to search).
Thanks
Shane

**Comment by dlech** [comment, marked as answer]
Bleak acts only as a BLE central, not a peripheral. There is a similar project https://github.com/kevincar/bless that does the opposite.

**Reply by shanedevane** [reply]
awesome thanks for the reply
